service/oceancd/verificationProvider.go

```diff
@@ -0,0 +1,510 @@
+}
+
+func (s *ServiceOp) CreateVerificationProvider(ctx context.Context, input *CreateVerificationProviderInput) (*CreateVerificationProviderOutput, error) {
+	r := client.NewRequest(http.MethodPost, "/ocean/cd/verificationProvider")
+	r.Obj = input
+
+	resp, err := client.RequireOK(s.Client.DoOrg(ctx, r))
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+
+	gs, err := verificationProvidersFromHttpResponse(resp)
+	if err != nil {
+		return nil, err
+	}
+
+	output := new(CreateVerificationProviderOutput)
+	if len(gs) > 0 {
+		output.VerificationProvider = gs[0]
+	}
+
+	return output, nil
+}
+
+func (s *ServiceOp) ReadVerificationProvider(ctx context.Context, input *ReadVerificationProviderInput) (*ReadVerificationProviderOutput, error) {
+	path, err := uritemplates.Expand("/ocean/cd/verificationProvider/{name}", uritemplates.Values{
+		"name": spotinst.StringValue(input.Name),
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	r := client.NewRequest(http.MethodGet, path)
+	resp, err := client.RequireOK(s.Client.DoOrg(ctx, r))
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+
+	gs, err := verificationProvidersFromHttpResponse(resp)
+	if err != nil {
+		return nil, err
+	}
+
+	output := new(ReadVerificationProviderOutput)
+	if len(gs) > 0 {
+		output.VerificationProvider = gs[0]
+	}
+
+	return output, nil
+}
+
+func (s *ServiceOp) UpdateVerificationProvider(ctx context.Context, input *UpdateVerificationProviderInput) (*UpdateVerificationProviderOutput, error) {
+	path, err := uritemplates.Expand("/ocean/cd/verificationProvider/{name}", uritemplates.Values{
+		"name": spotinst.StringValue(input.VerificationProvider.Name),
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	// We do NOT need the Name anymore, so let's drop it.
+	input.VerificationProvider.Name = nil
+
+	r := client.NewRequest(http.MethodPut, path)
+	r.Obj = input
+
+	resp, err := client.RequireOK(s.Client.DoOrg(ctx, r))
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+
+	gs, err := verificationProvidersFromHttpResponse(resp)
+	if err != nil {
+		return nil, err
+	}
+
+	output := new(UpdateVerificationProviderOutput)
+	if len(gs) > 0 {
+		output.VerificationProvider = gs[0]
+	}
+
+	return output, nil
+}
+
+func (s *ServiceOp) PatchVerificationProvider(ctx context.Context, input *PatchVerificationProviderInput) (*PatchVerificationProviderOutput, error) {
+	path, err := uritemplates.Expand("/ocean/cd/verificationProvider/{name}", uritemplates.Values{
+		"name": spotinst.StringValue(input.VerificationProvider.Name),
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	// We do NOT need the Name anymore, so let's drop it.
+	input.VerificationProvider.Name = nil
+
+	r := client.NewRequest(http.MethodPatch, path)
+	r.Obj = input
+
+	resp, err := client.RequireOK(s.Client.DoOrg(ctx, r))
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+
+	gs, err := verificationProvidersFromHttpResponse(resp)
```

Comment:
use meaningful name for variable instead of gs

Reply:
changed to vp